hikari/channels.py

```diff
@@ -1224,16 +1265,22 @@ class TextableGuildChannel(GuildChannel, TextableChannel):
 
 
 @attr.define(hash=True, kw_only=True, weakref_slot=False)
-class GuildCategory(GuildChannel):
+class GuildCategory(PermissibleGuildChannel):
     """Represents a guild category channel.
 
     These can contain other channels inside, and act as a method for
     organisation.
     """
 
+    parent_id: None
```

Comment:
This will require an `attr.field` to prevent https://github.com/hikari-py/hikari/pull/982 from happening again.